projects/canopy/src/lib/primary-message/primary-message.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, HostBinding, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+  selector: 'lg-primary-message',
+  templateUrl: './primary-message.component.html',
+  styleUrls: ['./primary-message.component.scss'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Should this also have `changeDetection: ChangeDetectionStrategy.OnPush`? Not _exactly_ sure where we do and don't use that.

Reply:
yes, great point, i forgot it 👍

Reply:
Updated :)